autolift-cats/src/test/scala/autolift/cats/LiftExistsTest.scala

```diff
@@ -0,0 +1,35 @@
+package autolift.cats
+
+import cats._
+import cats.implicits._
+
+//import autolift._
+import autolift.Cats._
+
+class LiftExistsTest extends BaseSpec {
+	case class Bar[A](a: A)
+
+	implicit val fn = new Functor[Bar]{
+		def map[A, B](fa: Bar[A])(f: A => B) = Bar(f(fa.a))
+	}
+
+	"liftExists" should "work on a List" in{
+		val out = List(1, 2, 3).liftExists(even)
+
+		same[Boolean](out, true)
+	}
+
+	"liftExists" should "work on a List[Option]" in{
+		val in = List(Option(1), None, Option(3))
+		val out: List[Boolean] = in liftExists even
```

Comment:
Ah, there is a limitation to export-hook due to the nature of the dependent types. You'll have to add the following or some variation of it to your reexport location:
```scala
  implicit def mkLE[Obj, Fn](implicit lift: CatsLiftExists[Obj, Fn]): CatsLiftExists.Aux[Obj, Fn, lift.Out] = lift
```